migrations/20240621000000_workflow_tries.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex) {
+    return knex.schema.withSchema('dbos')
+        .table('workflow_status', function(table) {
+
+            table.bigInteger('workflow_retries')
```

Comment:
Maybe it's more accurate to call it `recovery_attempts`? Because we only increase this counter on auto recovery but not on user generated retries (invoke a workflow with the same UUID multiple times).

Reply:
Agreed, I'll fix it

Reply:
I think this is incremented any time we call `internalWorkflow()`, so it is both on the "normal" retry path and the recovery path

Reply:
Nope, only on the recovery path